SWMessages/Source/SWMessageView.swift

```diff
@@ -218,6 +231,21 @@ public class SWMessageView :UIView , UIGestureRecognizerDelegate {
         fatalError("init(coder:) has not been implemented")
     }
     
+    public override func drawRect(rect: CGRect) {
+        super.drawRect(rect)
+        let options = SWMessageView.styleForMessageType(notificationType)
```

Comment:
Style as property wouldbe much better solution.

About maintain those changes in 3.0 branch. We can use cherrypick and then adjust syntax to 3.0.